tox.ini

```diff
@@ -1,6 +1,13 @@
 [tox]
-envlist = py27, py33, py34, py35, py36
+envlist = {py27,py33,py34,py35,py36}-{pytest,behave}
```

Comment:
I'll have to check if && is possible, I thought it was impossible to chdir in `tox.ini` (only with `changedir=`)